tolstoy/src/syncer.rs

```diff
@@ -413,13 +413,86 @@ impl RemoteClient {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use std::borrow::Borrow;
     use std::str::FromStr;
 
+    use edn;
+
+    use mentat_db::debug::{TestConn};
+
     #[test]
     fn test_remote_client_bound_uri() {
         let user_uuid = Uuid::from_str(&"316ea470-ce35-4adf-9c61-e0de6e289c59").expect("uuid");
         let server_uri = String::from("https://example.com/api/0.1");
         let remote_client = RemoteClient::new(server_uri, user_uuid);
         assert_eq!("https://example.com/api/0.1/316ea470-ce35-4adf-9c61-e0de6e289c59", remote_client.bound_base_uri());
     }
+
+    #[test]
+    fn test_add() {
```

Comment:
Maybe slim this down or do something a little more Tolstoy-esque for the test?  Your call, since it's clearly temporary.

Reply:
Maybe make one of your data types have `to_edn` and use `assert_matches!` on it?

Reply:
I will as I start landing other things. This is just to make sure everything glues together.